app/services/PayeAtsService.scala

```diff
@@ -48,4 +56,16 @@ trait PayeAtsService {
        }
      }
   }
+
+  private def sendAuditEvent(taxYear: Int)(implicit hc: HeaderCarrier, request: PayeAuthenticatedRequest[_]) = {
+    auditService.sendEvent(
+      auditType = AuditTypes.Tx_SUCCEEDED,
+      details = Map(
+        "userId" -> request.userId,
+        "userNino" -> request.nino.nino,
+        "taxYear" -> taxYear.toString,
+        "time" -> new Date().toString
```

Comment:
The time stamp is stored when it logs anyway, do we need this field?

Reply:
Yeah completely agree Timestamp is by default there in the logs so we don't need this time field.
Going to remove it.

Reply:
This was requested by the analyst building the dashboard, as it makes it equivalent to the payload for the SA part of the service, but I agree in this case it should be clarified.